ui/app/AppLayouts/Profile/Sections/Ens/SetPubKeyModal.qml

```diff
@@ -69,7 +69,13 @@ ModalPopup {
             AccountSelector {
                 id: selectFromAccount
                 accounts: walletModel.accounts
-                selectedAccount: walletModel.currentAccount
+                selectedAccount: {
+                    const currAcc = walletModel.currentAccount
+                    if (currAcc.walletType !== Constants.watchWalletType) {
+                        return currAcc
+                    }
+                    return null
+                }
```

Comment:
This code is duplicated a lot. Can we put it directly in `AccountSelector` instead?

Reply:
Not really unfortunately. Only when `selectedAccount`  of the `AccountSelector` is set to `walletModel.currentAccount` can we use this block of code, which is maybe about half the use cases of `AccountSelector`, so it wouldn't make sense to put it in `AccountSelector` as a default value. We could maybe add an option like `useCurrentAccountAsSelected` to `AccountSelector`, but it doesn't really feel right.

Reply:
>  which is maybe about half the use cases of AccountSelector

But  for the other half of cases, we override `selectedAccount` anyway right? So the `walletModel.currentAccount` would only be used when it is wanted. 

Maybe I'm not clear, I'll try to reformulate.

If we put that code block as a default, it should make sense since most AccountSelectors default to the currentAccount. The other cases when we don't want that, we just override `selectedAccount`, which we already do, so the default value of `walletModel.currentAccount` would be overwritten.

Reply:
I see what you mean. I do understand, however my personal opinion is that the default behaviour is already selecting the first account in the list of accounts in the model. I think that's a more intuitive default `selectedAccount` than `walletModel.currentAccount`. However, that is my own opinion, so definitely doesn't mean it is right.